docusaurus.config.js

```diff
@@ -25,7 +25,8 @@ module.exports = {
     announcementBar: {
       id: 'start',
       content:
-        '⭐️ If you like KubeVela, give it a star on <a target="_blank" rel="noopener noreferrer" href="https://github.com/kubevela/kubevela">GitHub</a>! ⭐️',
+        '⭐️ If you like KubeVela, give it a star on <a target="_blank" rel="noopener noreferrer" href="https://github.com/kubevela/kubevela">GitHub</a>!⭐️' +
+        '<span style="color:#1b58f4">KubeVela v1.4.8 is already released.(2022-07-20)</span> <a target="_blank" rel="noopener noreferrer" href="https://github.com/kubevela/kubevela/releases/v1.4.8">Read Release Note</a>',
```

Comment:
can it changed automatically?

Reply:
Maybe we could add a bot, and commit a PR after creating a release.